x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/expression/function/scalar/FunctionTestUtils.java

```diff
@@ -39,8 +45,9 @@ public Combinations(int n, int k) {
 
         @Override
         public Iterator<BitSet> iterator() {
-            return new Iterator<BitSet>() {
+            return new Iterator<>() {
```

Comment:
This auto-formatting is kinda of annoying and distracting - make sure to enable auto-formatting on the lines you actually change not the whole file.
It's fine to improve the code however I would that in a separate PR.

Reply:
It was done manually, on purpose, it's not auto-formatting just a small code style improvement.
If you agree, I think I'd keep it since it's quite small.